test-clients/src/main/java/com/hedera/services/bdd/spec/queries/contract/HapiGetContractInfo.java

```diff
@@ -132,7 +132,7 @@ protected void assertExpectationsGiven(HapiApiSpec spec) throws Throwable {
 		var actualTokenRels = actualInfo.getTokenRelationshipsList();
 		assertExpectedRels(contract, relationships, actualTokenRels, spec);
 		assertNoUnexpectedRels(contract, absentRelationships, actualTokenRels, spec);
-		Assertions.assertEquals(ByteString.copyFromUtf8(expectedLedgerId), actualInfo.getLedgerId());
+		Assertions.assertEquals(rationalize(expectedLedgerId), actualInfo.getLedgerId());
```

Comment:
It's probably better to have these `ledgerId` assertions in the `*GetInfo` queries be off by default; otherwise, every time we want to run a query against a non-dev network, it will fail. 😂